kotlin-format/kotlin-format.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+KOTLIN_FORMAT_DIR="$(dirname "${BASH_SOURCE[0]:-${(%):-%x}}")"
+exec java --add-opens java.base/java.lang=ALL-UNNAMED -Xmx512m -jar "${KOTLIN_FORMAT_DIR}/kotlin-format.jar" "$@"
```

Comment:
do you know if the `--add-opens` stuff is required? If it's not then I'd prefer to remove it